src/umpire/resource/DefaultMemoryResource.hpp

```diff
@@ -20,7 +20,9 @@ namespace resource {
  * allocate and deallocate memory.
  */
 template <typename _allocator>
-class DefaultMemoryResource : public MemoryResource {
+class DefaultMemoryResource : 
+  public MemoryResource
+{
```

Comment:
```suggestion
class DefaultMemoryResource : public MemoryResource {
```

Reply:
(Again, being picky, but I think most of our other inheritance lines like this have been structured this way instead)